Dockerfile

```diff
@@ -19,50 +19,27 @@ COPY MATScommon /MATScommon
 RUN bash ${SCRIPTS_FOLDER}/build-meteor-bundle.sh
 
 
-# Install OS build dependencies
-FROM node:14-alpine AS native-builder
-
-ENV APP_FOLDER=/usr/app
```

Comment:
These three environmental variables here look like they are removed, but then they are referenced later in code that still exists. Was this intentional?

Reply:
(Lines 25-27)

Reply:
Good eye. It was intentional. We were originally doing a 3-part multistage Docker build in this Dockerfile. You can tell how many "parts" there are to a multistage build by counting the number of `FROM` lines - those indicate a new container is being started.

Since we're no longer using Alpine we didn't need to do the second part of the build where we recompiled our native dependencies for that OS. I've gone ahead and removed the entirety of that step. However, variables with the same name are later defined and used in the third build step. It didn't come across well in GitHub's diff view but you can see the definitions here: https://github.com/NOAA-GSL/MATS/blob/f54a710cdade1c9f5a3c99a999b75067f7a30469/Dockerfile#L44-L48. 

In hindsight, it probably would've been good to have named those variables something different in the intermediate steps in order to cut down on confusion and make it clear which step they belonged to.